doc/sphinxext/mock_gui_toolkits.py

```diff
@@ -113,20 +113,13 @@ def getapi(*args):
 
 
 class MyWX(MagicMock):
-    class Panel(object):
-        pass
-
-    class ToolBar(object):
-        pass
-
-    class Frame(object):
-        pass
-
+    Frame = Panel = Printout = ToolBar = type("", (), {})
     VERSION_STRING = '2.9'
 
 
 def setup(app):
     sys.modules['cairocffi'] = MyCairoCffi()
+    sys.modules['cairocffi'].__name__ = 'cairocffi'
```

Comment:
Shouldn't this go on the `MyCairoCffi` class?

Reply:
indeed, that works too